core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -429,7 +429,7 @@ enum class UniqueType(
     UnitUpgradeCost("[relativeAmount]% Gold cost of upgrading", UniqueTarget.Unit, UniqueTarget.Global),
 
     // Gains from battle
-    DamageUnitsPlunder("Earn [amount]% of the damage done to [combatantFilter] units as [civWideStat]", UniqueTarget.Unit, UniqueTarget.Global),
+    DamageUnitsPlunder("Earn [amount]% of the damage done to [mapUnitFilter] units as [civWideStat]", UniqueTarget.Unit, UniqueTarget.Global),
```

Comment:
Doubly so here, as I think (but haven't checked) this actually affects some units in the built in rulesets